sasu-drooz/Domoticz-Zigate
# RC 6.3.006 (#1339)

* fix resetSwitchSelectorPushButton for Button_3
* integrate new zigpy packet-api phase1 (#1324)
* aligning method signatures
* merging upstream with packet-api included
* update requirements level
* update the zigpy radio module version check
* enable source_routing for all coordinators
* merge upstream - fix ZDO deserialization
* improve log message on DeliveryError
* set a specific debug flag for DeliveryError on unicast messages
* zigpy 51.4
* Hot-fix to prevent plugin not starting in case of uninitialized Coordinator and wrong backup file (#1330)
* better error management and return None in the backup file is not in Json format, or un-readable
* WebUI translation update
* Integrating ST30 manufactured by Orvibo Temperature & Humidity Sensor
* Adapt plugin to the changes of ZLinky v11 (#1332)
* handle ZLinky firmware which now send SINSTS and SINSTS1 on different attributes depending on Tri-phased or Mono
* correct /zigpy-raw-command API
* User the new Domoticz command Open , Close for blinds ( Domoticz 2022.2 )
* Implement unbind_response_command #1302 (#1333)
* Addressing #1302 and implementing unbind_response_command
* Addressing #1302 and implementing unbind_response_command
* Merge latest zigpy development (#1331)
* merging the latest update from zigpy on Only enable APS ACKs for packets without implicit delivery notification
* correct the habndling of AckMode from WebUI with the ackIsDisabled parameter
* inverse expect_reply settings due to the zigpy PR https://github.com/zigpy/zigpy/pull/1085
* Source Routing parameter is now avaliable for all Zigpy radio libs
* add logging message at startup
* register needed clusters when registering endpoints
* refactor endpoint registration in a more generic way
* accept up to a load of 8 request to process a callback, otherwise wait for the next one
* refactor how to handle setpoint for VACT Wiser
* set extended_timeout to True in order to get commands reaching End Devices
* set the ZNP CONF_PREFERED_ENDPOINT1 to False if there is at least one specific Ep to be registered
* add Cluster 0xe002
* TS130F-1dd.... seems to be used as a Blinds or as a Vanne
* use Curtain instead of Blind
* when preparing the RAW Command, provide context information on the target device, if it is a RX on Idle or not
* In case we send a request to a device which do not Rx on Idle, then make sure to set the extended_timeout to True
* in case where device do not exist just return rx on idle True
* remove the rule which was setting Wiser Ep requested when enableSchneiderWiser is on
* Move level with On/off when group command. (#1336)
* Pipiche38/group/move level onoff (#1338)
* prevent overwriting the value passed to MajDomoDevice
* upgrade zigpy-deconz version (#1340)
* makes source_routing disabled by default
* Merge upstream bellows 0.43.3 (#1343)
- Allow disabling the bellows UART thread
- Allow configuration options to be disabled
- make ACK dependents on packet.tx_options
* Wiser VACT improvement on Zigpy (#1335)
* give the possibility to delay the setpoint raw command, and waits for a call back to execute it, so we will no send a command which is going to fail because the end device do not listen
* update Setpoint if it is not matching what has been reported in a 12 minutes window
* Handle Open and Close command from Domoticz for Blinds
* fix way of implementing zclGroup commands.
* fix #1344
* enable Decode8502 for OTA Page request
* fixing TypeError: unsupported operand type(s) for //: 'str' and 'str'
* makes Group Level command with or without On/Off customizable based on GroupLevelWithOnOff parameter
* refactor GroupLevelWithOnOff
* fix issue when RxOnIdle is not set
* disable SourceRouting and disable ForceAPSAck
* add some WebUI translation
